docs/practices/fast_builds.md

```diff
@@ -1,34 +1,36 @@
 # Fast Builds
 
 nearcore is implemented in Rust and is a fairly sizable project, so it takes a
-while to build. This chapter collects various tips to make the process faster.
+while to build. This chapter collects various tips to make the development
+process faster.
 
 Optimizing build times is a bit of a black art, so please do benchmarks on your
-machine to verify that the improvement work for you. Changing some configuration
+machine to verify that the improvements work for you. Changing some configuration
 and making some type, which prevents it from improving build times is an
 extremely common failure mode!
```

Comment:
I’m not quite sure what the last sentence is trying to say here. What does "and making some type” mean and the way it affects build times doesn't really connect.

Reply:
X_X that was supposed to be "some typo"